mscore/file.cpp

```diff
@@ -2309,10 +2315,21 @@ Score::FileError readScore(MasterScore* score, QString name, bool ignoreVersionE
       if (synti)
             score->setSynthesizerState(synti->state());
 
-      if (suffix == "mscz" || suffix == "mscx") {
+      auto read = [](MasterScore* score, const QString& name, bool ignoreVersionError, bool imported = false)->Score::FileError {
```

Comment:
Not exactly. I suggested writing a method similar to `importMusicXml` or `importCompressedMusicXml`, etc. Then, you just assign the reference to the method to the `mscz,` and `mscx,` extensions as it is done in Line 2351 and below.

Reply:
What about the `ignoreVersionError` parameter?